cloud_experts_osd_tutorials/cloud-experts-osd-create-new-limit-egress.adoc

```diff
@@ -0,0 +1,385 @@
+
+----
++
+[IMPORTANT]
+====
+If there is not a matching rule that allows the traffic, it will be blocked by the firewall. To allow access to other resources, such as internal networks or other external endpoints, create additional rules with a priority of less than 1000. For more information on how to create firewall rules, see  link:https://cloud.google.com/firewall/docs/use-network-firewall-policies[Use global network firewall policies and rules].
+====
+
+[id="creating-osd-gcp-cluster_{context}"]
+== Creating your cluster
+You are now ready to create your {product-title} on {GCP}  cluster. For more information, see xref:../osd_gcp_clusters/creating-a-gcp-cluster-with-workload-identity-federation.adoc#osd-creating-a-cluster-on-gcp-with-workload-identity-federation[Creating a cluster on GCP with Workload Identity Federation authentication].
+
+[id="deleting-osd-gcp-cluster_{context}"]
+== Deleting your cluster
+To delete your cluster, see xref:../osd_gcp_clusters/osd-deleting-a-cluster-gcp.adoc#osd-deleting-a-cluster-gcp[Deleting an OpenShift Dedicated cluster on GCP].
+
+
+[id="cleaning-resources-osd-gcp-cluster_{context}"]
+== Cleaning up resources
+
+To prevent ongoing charges, after you delete your cluster you must manually delete the Google Cloud networking infrastructure you created as part of this tutorial. Deleting the cluster will not automatically remove these underlying resources. You can clean up these resources using a combination of gcloud CLI commands and actions within the Google Cloud console.
+
+Before you begin the process of cleaning up the the resources you created for this tutorial, run the following commands and complete any prompts.
+
+. To authenticate your identity run the following command:
++
+[source,terminal]
+----
+$ gcloud init
+----
++
+. To log in to your Google Cloud account, run the following command:
++
+[source,terminal]
+----
+$ gcloud auth application-default login
+----
++
+. To log in to the OpenShift Cluster manager CLI tool, run the following command:
++
+[source,terminal]
+----
+$ ocm login --use-auth-code
+----
+
+You are now ready to clean up the resources you created as part of this tutorial. To respect resource dependencies, delete them in the reverse order of their creation.
+
+. Navigate to the Google Cloud console and remove the firewall policy's association with the VPC.
+. Delete the firewall policy's association with the VPC by running the following command:
++
+[source,terminal]
+----
+$ gcloud compute network-firewall-policies associations delete \
+    --firewall-policy=${prefix} \
+    --network=${prefix}-vpc \
+    --global-firewall-policy
+----
```

Comment:
These two steps are identical, except one is performed through the console while the other is performed using gcloud commands. I think for consistency, we should stick to gcloud commands only.